caffe2/utils/math/reduce.h

```diff
@@ -19,6 +96,7 @@ CAFFE2_API void Moments(
     Context* context);
 
 } // namespace math
+
```

Comment:
is this expected?

Reply:
Yes, this } is the end of namespace math.